### ISSUE TITLE
Suggest `highlight` class change to `htHighlight` due to app conflicts

### ISSUE THREAD

**Issue by phc284** [handsontable/handsontable#5573]
### Description
Class name `highlight` is very generic. Other classnames already in handson table start with `ht` such as `htRight`, `htMiddle`, `htNumeric`, etc. People with large applications which already have a `.highlight` class will have to refactor due to this.

Suggestion is to change the cell's `highlight` class to `htHighlight` or similar.

### Steps to reproduce
1. Inherit to api

### Demo

https://jsfiddle.net/handsoncode/8ffpsqt6/

### Your environment
* Handsontable version: 6.1.1
* Browser Name and version: Chrome/Firefox
* Operating System: MacOS

**Comment by AMBudnik** [comment]
I agree. We should use classes names that are associated with the product. As we use `htInvalid` or `htCommentCell` we should change it to `htHighlight`
